bingo/EA/RandomIndividualVariation.py

```diff
@@ -0,0 +1,50 @@
+from bingo.Base.Variation import Variation
+from bingo.MultipleValues import MultipleValueGenerator
+from bingo.Util.ArgumentValidation import argument_validation
+
+class RandomIndividualVariation(Variation):
```

Comment:
lets use the (slightly) more expressive name AddRandomIndividualsVariation

Reply:
Well will do that :).